.github/workflows/_build_test_upload.yml

```diff
@@ -62,22 +62,46 @@ jobs:
           - macos-latest
           - ubuntu-latest
           - windows-latest
+          - macos-m1-12
```

Comment:
Is this an internal Meta self-hosted runner or does GitHub Actions finally have public M1 runners?

Reply:
Our self-hosted runner